src/account/account_repository.hpp

```diff
@@ -27,6 +27,8 @@ class AccountRepository {
 	virtual bool loadBySession(const std::string &email, AccountInfo &acc) = 0;
 	virtual bool save(const AccountInfo &accInfo) = 0;
 
+	virtual bool getCharacterByNameAndAccountId(const uint32_t &id, const std::string &name) = 0;
```

Comment:
maybe, we can use the function name as parameter order like : getCharacterByAccountIdAndName